coral/src/app/pages/topics/schema-request.test.tsx

```diff
@@ -23,7 +23,7 @@ const mockGetTopicNames = getTopicNames as jest.MockedFunction<
   typeof getTopicNames
 >;
 
-describe("SchemaRequest", () => {
+describe("SchemaRequestModel", () => {
```

Comment:
I think that sneaked itself in :D

Reply:
Search and replace woes haha [3b5e030](https://github.com/aiven/klaw/pull/843/commits/3b5e030a7baee9b6a70c6c8f77b8f5a85b36ea08)